### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by Artemis21**
- Limit line length to 80.
- Refactor constants.
- Consistently type functions.
- Don't log tokens to stdout.
- Use wrapper function to parse callback data.
- Use consistent quotes.
- Use discord.py utility to get user avatar URLs.
- Allow the data file to be initially not present.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -169,56 +173,50 @@ async def check(ctx: commands.Context):
...
         )
-        
+
 
 @bot.event
 async def on_member_join(member: discord.Member):
     try:
         account = await polympics_client.get_account(member.id)
-    except Exception:
+    except polympics.PolympicsError:
```
Comment (Legorooj) [inline main.py]: This won't work - `polympics.PolympicsError`s aren't what I'm trying to catch. I get an aiohttp error if the user doesn't exist.

File: main.py [hunk 2]
```diff
@@ -169,56 +173,50 @@ async def check(ctx: commands.Context):
...
-            avatar_url = f'https://cdn.discordapp.com/avatars/{account.id}/{after.avatar}.{ext}'
-        else:
-            avatar_url = account.avatar_url
-        
         await polympics_client.update_account(
-            account, name=after.name, discriminator=after.discriminator, avatar_url=avatar_url
+            account,
+            name=after.name,
+            discriminator=after.discriminator,
+            avatar_url=str(after.avatar_url)
```
Comment (Legorooj) [inline main.py]: I wasn't using avatar_url explicitly, because it returns the avatar sized.

Comment (Artemis21) [reply main.py]: Good point. The issue with your implementation was that it didn't handle default avatars.

Comment (Artemis21) [reply main.py]:
```suggestion
            avatar_url=str(after.avatar_url).split('?')[0]
```
Bit hacky but how's this?

File: main.py (hunk 1 above)
Comment (Artemis21) [reply main.py]: Hmm, sounds like a wrapper bug.

File: main.py (hunk 2 above)
Comment (Legorooj) [reply main.py]: Looks good.

File: main.py (hunk 1 above)
Comment (Artemis21) [reply main.py]: Oh well. Could also be a server bug, I didn't test exception handling well.

**Pull request merged**